dist/index.html

```diff
@@ -595,8 +595,8 @@ <h4 class="text-c100 font-bold">
         </div>
       </div>
     </section>
-
-    <section class="sponsors">
+    <hr class="hr-thin" />
```

Comment:
Great work. You can style the element  with class`sponsers`  instead of using `hr`

Reply:
thanks for your help and precious feedbacks